content/docs/developer/custom-networks/_index.md

```diff
@@ -4,22 +4,52 @@ linkTitle: "Custom Network"
 weight: 200
 description: >
   Custom Autonity networks
-draft: true
+draft: false
 ---
 
-- To connect your node to an existing Autonity network other than `bakerloo` or `piccadilly`, you will need the network's:
-  - bootnode file `static-nodes.json`
-  - genesis file `genesis.json`
+To connect your node to a custom Autonity network, you will need the network's:
+  - bootnodes, typically given in a  `static-nodes.json` file or specified when running the node with the `--bootnodes` command-line option.
+  - genesis configuration file,  `genesis.json`.
 
-  See the [Networks](/networks/) section for file download links.
+  See [Local Autonity Network configuration](/reference/genesis/#local-autonity-network-configuration) in the [Genesis](/reference/genesis/) reference for how to create these files.
 
-{{% alert title="Note" %}}Note that the client provides command-line flag options for connecting to the various Autonity testnets. If you specify the testnet flag, then neither genesis nor bootnode files are required: the client will use the flag to set genesis and bootstrap configuration. {{% /alert %}}
+{{% alert title="Note" %}}Note that the client provides command-line flag options for connecting to the various Autonity testnets: `--bakerloo` and `--piccadilly`. If you specify the testnet flag, then neither genesis nor bootnode files are required: the client will use the flag to set genesis and bootstrap configuration. {{% /alert %}}
```

Comment:
So if you use one of the flags, `static-nodes.json` and `genesis.json` _aren't required_. That's ambiguous. Are they ignored, or if they're present, do they override the flags?

Reply:
If both genesis & static-nodes **and** a testnet flag is specified in the run command, the testnet flag takes precedence and an error is thrown. This happens if re-starting the local devnet or starting it for the first time. 

For example, if running a local dev testnet on localhost the `--piccadilly` flag is added so:

```console
ubuntu@vps-c7c3e8c7:~/TEST/develop$ ./autonity/build/bin/autonity --datadir ./autonity-chaindata  --genesis ./genesis.json  --http  --http.api aut,eth,net,tendermint,txpool,web3,admin  --ws  --ws.api aut,eth,net,personal,tendermint,txpool,web3,admin  --networkid 65110000 --nat extip:51.89.151.55 --metrics --pprof --http.port 8545 --ws.port 8546 --port 30303 --syncmode full --piccadilly ;
```

```console
Fatal: Failed to register the Autonity service: database contains incompatible genesis (have 2e5217f210d42c3ed2117a56be361c8cdb80a9dc4a99902330c716f058979043, new 712c50241a528508c86490fd883d75bcc3166a98147bfe13b51db0be8c5e5db6)
```

Where `new` is the `hash` of the Piccadilly genesis block minus the `0x` prefix:

>"hash": "0x712c50241a528508c86490fd883d75bcc3166a98147bfe13b51db0be8c5e5db6", 

```
ubuntu@vps-c7c3e8c7:~/TEST/autcli$ aut block get 0
{"baseFeePerGas": 1000000000, "committedSeals": [], "committee": [{"address": "0x31870f96212787d181b3b2771f58af2bed0019aa", "votingPower": "0x21e19e0c9bab2400000"}, {"address": "0x32f3493ef14c28419a98ff20de8a033cf9e6ab97", "votingPower": "0x21e19e0c9bab2400000"}, {"address": "0x36288c1f8c990fd66a1c5040a61d6f3ecf3a49c1", "votingPower": "0x21e19e0c9bab2400000"}, {"address": "0x6ebb5a45728be7cd9fe9c007add1e8b3daff6b3b", "votingPower": "0x21e19e0c9bab2400000"}, {"address": "0xac245af88265e72881cd9d21efb9ddc32e174b69", "votingPower": "0x21e19e0c9bab2400000"}, {"address": "0xb3a3808c698d82790ac52a42c05e4bcb3dfcd3db", "votingPower": "0x21e19e0c9bab2400000"}], "difficulty": 0, "extraData": "0x", "gasLimit": 30000000, "gasUsed": 0, "hash": "0x712c50241a528508c86490fd883d75bcc3166a98147bfe13b51db0be8c5e5db6", "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000", "miner": "0x0000000000000000000000000000000000000000", "mixHash": "0x63746963616c2062797a616e74696e65206661756c7420746f6c6572616e6365", "nonce": "0x0000000000000000", "number": 0, "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000", "proposerSeal": "0x", "receiptsRoot": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421", "round": "0x0", "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347", "size": 716, "stateRoot": "0x5cf9b155aff6d7a36cb648850831781997713195867fbb8a846d849ec447493b", "timestamp": 0, "totalDifficulty": 0, "transactions": [], "transactionsRoot": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421", "uncles": []}
```

Reply:
So, to remove the ambiguity, the note needs rewording, e.g.:

From:

>Note that the client provides command-line flag options for connecting to the various Autonity testnets: `--bakerloo` and `--piccadilly`. If you specify the testnet flag, then neither genesis nor bootnode files are required: the client will use the flag to set genesis and bootstrap configuration.

To:

>Note that the client provides command-line flag options for connecting to public Autonity testnets: `--bakerloo` and `--piccadilly`. The node will not run if you specify both genesis and bootnodes for a custom network **and** a testnet flag. The client will create a genesis block for the custom network's genesis configuration and the node's local store will then have an incompatible genesis with the testnet.

Reply:
Edit in commit b8be2c5c26eff531b83fb3ec8ecb53fc6c15c155

Someone specifying both is a hedge scenario, but  it's probably worth keeping  the note if it does happen.